modules/decima-ext-model-exporter/src/main/java/com/shade/decima/ui/data/viewer/model/BaseModelExporter.java

```diff
@@ -5,67 +5,64 @@
 import com.shade.decima.model.rtti.objects.RTTIReference;
 import com.shade.decima.model.rtti.registry.RTTITypeRegistry;
 import com.shade.decima.model.rtti.types.RTTITypeEnum;
-import com.shade.decima.ui.data.viewer.model.utils.Matrix3x3;
-import com.shade.decima.ui.data.viewer.model.utils.Matrix4x4;
-import com.shade.decima.ui.data.viewer.model.utils.Transform;
-import com.shade.decima.ui.data.viewer.model.utils.Vector3;
 import com.shade.platform.model.util.MathUtils;
 import com.shade.util.NotNull;
+import org.joml.*;
 
 import java.util.UUID;
 
 public class BaseModelExporter {
     @NotNull
-    protected static Transform worldTransformToTransform(@NotNull RTTIObject transform) {
+    protected static Matrix4dc worldTransformToMatrix(@NotNull RTTIObject transform) {
```

Comment:
There's no reason to return a constant variation because the caller owns the newly created instance, not the callee.